capstone/src/main/webapp/script.js

```diff
@@ -139,6 +139,7 @@ function precinctControl(controlDiv, map) {
     document.getElementById('chart-title').textContent =
      precinct + ' Police Sentiment';
     loadCharts();
+    loadWordcloud();
```

Comment:
Loading the cloud here may lead to increased latency here. Alternative solution would be to load the cloud only when the top associations panel is selected. This would reduce the time it takes for the side panels to render when a user selects a precinct.